CRM/Activity/Form/Activity.php

```diff
@@ -134,6 +134,10 @@ class CRM_Activity_Form_Activity extends CRM_Contact_Form_Task {
 
   public $_entityTagValues;
 
+  public $_caseStatus;
+
+  protected $_defaultCaseStatus;
+
```

Comment:
CiviCase is full of fun™ stuff. ChangeCaseStatus is not actually an Activity object - note it does not extend - and gets instantiated and called from "regular" Activity. Like this fun™ code: https://github.com/civicrm/civicrm-core/blob/ea39ef4337bcdc86e3f8e9c94d6bc52e4a1faf32/CRM/Activity/Form/Activity.php#L812-L817. So the `$form->_XXX` is actually a plain activity object.

Reply:
Ah, I was looking for something like that, but was not successful in tracking it down. 

I think `_caseStatus` and `_defaultCaseStatus` could do with a `@see CRM_Case_Form_Activity_ChangeCaseStatus` to avoid future confusion.